.circleci/config.yml

```diff
@@ -4,14 +4,15 @@ jobs:
   validate-changes:
     docker:
       # pre-commit package is included since 20.10
-      - image: ubuntu:20.10
+      - image: ubuntu:22.04
```

Comment:
Why upgrade Ubuntu?
We're going to be swapping out Circle CI with Github Actions in the very near future.

Reply:
Nevermind, I see your commit comment from 7c25bec.

[May I ask you to share the result of a brief test with this branch image?](https://hub.docker.com/layers/gitpod/workspace-full/branch-prs-upgrade-packages/images/sha256-c88f51fae03baa8d0bb1be74dd3b26e242dfeb61031380e24405e04b2d488f71?context=explore)

Reply:
22.04 is not even feature locked yet. I would be curious to see why it was failing with 20.10.

Reply:
> why it was failing with 20.10

The errors as seen in [build](https://app.circleci.com/pipelines/github/gitpod-io/workspace-images/1531/workflows/c261fd8d-3a09-4e73-9490-89668b093bd1/jobs/7462) are 


```
E: The repository 'http://security.ubuntu.com/ubuntu groovy-security Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://archive.ubuntu.com/ubuntu groovy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://archive.ubuntu.com/ubuntu groovy-updates Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://archive.ubuntu.com/ubuntu groovy-backports Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details
```
which mean that the Release file was removed for some dependency/(ies) for ubuntu 20.10. To fix this we need to remove ppa and then update the image which seems unnecessary effort.

Reply:
does the same error happen with 21.10 (it is not LTS, but at least it is a stable version)

Reply:
If this is only impacting the builder runner, I think it's fine to use 22, Circle CI is going to have a short life, we'll be replacing it with Github Actions soon.

@princerachit , if you `docker run -it --rm gitpod/workspace-full:branch-prs-upgrade-packages` and then `lsb_release -a`, what does the output look like? If it's still Ubuntu 20, we're good. If it's somehow upgraded to Ubuntu 22, let's reconsider.

Reply:
I see what you mean, @sagor999 . In looking [here](https://wiki.ubuntu.com/Releases), 22 isn't even on the list! :thinking: 

Yeah, using 21.10 makes more sense. Thanks for the add! :pray:

Reply:
```
➜  ~ docker run -it gitpod/workspace-base:branch-prs-upgrade-packages
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
gitpod ~ $ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.3 LTS
Release:	20.04
Codename:	focal
gitpod ~ $ exit
➜  ~ docker run -it gitpod/workspace-full:branch-prs-upgrade-packages
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
gitpod ~ $ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```